apps/openmw/mwgui/tradewindow.hpp

```diff
@@ -65,8 +65,10 @@ namespace MWGui
 
             int mItemToSell;
 
-            int mCurrentBalance;
-            int mCurrentMerchantOffer;
+            bool mIsPlayerSelling;
+            bool mIsMerchantBuying;
```

Comment:
Yes, I know - it says so in the pull request description. I'm just wondering if there's a less invasive way of implementing this. For example, by only changing these two lines:

<pre>
diff --git a/apps/openmw/mwgui/tradewindow.cpp b/apps/openmw/mwgui/tradewindow.cpp
index c084ec2..791f8df 100644
--- a/apps/openmw/mwgui/tradewindow.cpp
+++ b/apps/openmw/mwgui/tradewindow.cpp
@@ -477,14 +477,14 @@ namespace MWGui
         if (mCurrentBalance == INT_MAX || mCurrentBalance == INT_MIN+1)
             return;
         if(mCurrentBalance<=-1) mCurrentBalance -= 1;
-        if(mCurrentBalance>=1) mCurrentBalance += 1;
+        else mCurrentBalance += 1;
         updateLabels();
     }
 
     void TradeWindow::onDecreaseButtonTriggered()
     {
-        if(mCurrentBalance<-1) mCurrentBalance += 1;
-        if(mCurrentBalance>1) mCurrentBalance -= 1;
+        if(mCurrentBalance<=-1) mCurrentBalance += 1;
+        else mCurrentBalance -= 1;
         updateLabels();
     }
 
</pre>

And FWIW, setting the offer to 0 was already possible by using the text entry box.